Cherish/Utility/Color.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  Color.swift
+//  Cherish
+//
+//  Created by 이가은 on 2022/08/03.
+//
+
+import SwiftUI
+
+extension Color {
+    // Main
+    static let darkGreen = Color("darkGreen")
+    static let lightGreen = Color("lightGreen")
+    static let backgroundGreen = Color("backgroundGreen")
+    // Text
+    static let dateText = Color("dateText")
+    static let defaultText = Color("defaultText")
```

Comment:
```suggestion
    enum Main {
        static let darkGreen = Color("darkGreen")
        static let lightGreen = Color("lightGreen")
        static let backgroundGreen = Color("backgroundGreen")
    }
    // Text
    static let dateText = Color("dateText")
    static let defaultText = Color("defaultText")
```

enum을 통해서 구분한다면 어떨까요 ? 사용하는 쪽에서도 `Color.Main.darkGreen`이면 이해하기 쉬울 것 같아요

Reply:
이 부분에 대해서 고민을 해봤었는데, 아직 Color가 main과 text로 분명하게 나뉜 게 아니라서, 후에 fix가 되면 수정해보려 합니다 ! (darkGreen도 text에 쓰일 때가 있어서요 🥹)